hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <odoo>
 
-    <record id="hr_timesheet_sheet_tree_simplified" model="ir.ui.view">
+    <record id="hr_timesheet_sheet_tree" model="ir.ui.view">
```

Comment:
@alexey-pelykh this is what you have to prevent to modify inside the same version, as inherited views will break with this ID change

Reply:
No known modules found in OCA

Reply:
100%, but we did a major module version change? That means something?